### ISSUE TITLE
Media support for Notes app [feature request]

### ISSUE THREAD

**Issue by eemantsal** [nextcloud/server#6768]
Do you plan to add media support for the notes? I've read that NC Notes app developers say it depends on you (your API, they say in their GitHub. I don't know exactly what it is, I'm just an user. Sorry for my ignorance); that when your API supports media attachments they will be able to implement media support too.
The support for multimedia content is a basic feature that almost every notes app includes (Evernote, Onenote, Keep, etc) since nowadays many people writes notes from their portable devices and adds photos, screenshots, short videos, etc. For instance, I have been doing some shopping this afternoon with my girlfriend, and she has thought about a lot of presents we could buy for a lot of people the next Christmas (yes, a foresighted woman she is; she knows I hate going shopping, hence took advantage of the occasion).
When we were having a look to some possible presents I had loved to write a note from my mobile like:

---
For dad:
[Photo]
Footnote with the shop's name, address, whatever.

For mother in law:
[Photo, or photo2]
Footnote with...

For brother:
[Photo, or photo2, or maybe photo3, he's been always so complicated...]
Footnote

Comment, maybe a quick audio record instead of stopping to type there in the middle of those commercial streets full of people and tourists.

---

With that reminding note, the next time I had to go shopping I'd have a clear reference and go directly for the products instead wasting hours looking this and that, like I always end doing; I could even send it to other person ini the family to do part of the work. x-D
Well, finally that's what I did, I took that note with that exact scheme, of course not in the NC notes mobile app since it doesn't support image embedding because NC Server doesn't, but in a chat with myself in Telegram. Is far from optimal but kind of works.

Taking notes from our portable devices and synchronize them, share them, edit them, review the pictures comfortably on our living room's telly, etc, is such a great progress that I think we all would benefit in some degree from a media capable notes app for our greatly loved Nextcloud platform that you guys make possible with your great work. I hope you'll consider media support for notes at least a quarter as useful as i do, and think about adding this feature in a proximate future.

Cheers.

**Comment by nickvergessen** [comment]
I think you have to report this to the notes app: https://github.com/nextcloud/notes/ ?

**Comment by eemantsal** [comment]
I thought so, so I first looked at  https://github.com/nextcloud/notes and searched into the issues, open and closed, to see if my suggestion had been already made, and I saw this comment from one of the developers:

> When the server app provides more APIs (like image support - or categories recently) - i will try to add those to the Android client as soon as possible.

https://github.com/nextcloud/notes/issues/90 (5th comment from the end, posted the 8 of May)

That's why I've thought that I should ask here instead. Sorry if I was wrong. You think that I should ask them even if they say it ulltimately depends on your APIs?
